pkgs/aws/amazon-ec2-spot-interrupter/registry.yaml

```diff
@@ -0,0 +1,20 @@
+packages:
+  - type: github_release
+    repo_owner: aws
+    repo_name: amazon-ec2-spot-interrupter
+    asset: ec2-spot-interrupter_{{trimV .Version}}_{{.OS}}_{{.Arch}}.tar.gz
+    description: The ec2-spot-interrupter is a simple CLI tool that triggers Amazon EC2 Spot Interruption Notifications and Rebalance Recommendations
+    replacements:
+      darwin: Darwin
+      linux: Linux
+    supported_envs:
+      - linux
+      - darwin
+    overrides:
+      - goos: linux
+        goarch: arm64
+        replacements:
+          arm64: armv6
```

Comment:
```suggestion
```

Reply:
https://github.com/aws/amazon-ec2-spot-interrupter/blob/ff44bb8991cc1e90f301542ef13148c09dac48ad/.goreleaser.yaml#L15

I think armv6 is different from arm64.

I confirmed we can't execute this on linux/arm64.

```console
shunsukesuzuki@lima-default:/Users/shunsukesuzuki/repos/src/github.com/aquaproj/aqua-registry$ ec2-spot-interrupter --help
FATA[0000] aqua failed                                   aqua_version=1.19.1 env=linux/arm64 error="it failed to start the process" program=aqua

shunsukesuzuki@lima-default:/Users/shunsukesuzuki/repos/src/github.com/aquaproj/aqua-registry$ aqua which ec2-spot-interrupter
/home/shunsukesuzuki.linux/.local/share/aquaproj-aqua/pkgs/github_release/github.com/aws/amazon-ec2-spot-interrupter/v0.0.8/ec2-spot-interrupter_0.0.8_Linux_armv6.tar.gz/ec2-spot-interrupter
shunsukesuzuki@lima-default:/Users/shunsukesuzuki/repos/src/github.com/aquaproj/aqua-registry$ /home/shunsukesuzuki.linux/.local/share/aquaproj-aqua/pkgs/github_release/github.com/aws/amazon-ec2-spot-interrupter/v0.0.8/ec2-spot-interrupter_0.0.8_Linux_armv6.tar.gz/ec2-spot-interrupter --help
bash: /home/shunsukesuzuki.linux/.local/share/aquaproj-aqua/pkgs/github_release/github.com/aws/amazon-ec2-spot-interrupter/v0.0.8/ec2-spot-interrupter_0.0.8_Linux_armv6.tar.gz/ec2-spot-interrupter: cannot execute binary file: Exec format error
```

Reply:
Thank you for pointing this out. I thought it was a typo.